src/amo/components/SiteNotices/index.js

```diff
@@ -30,7 +30,7 @@ type InternalProps = {|
 // We cannot use `sanitizeUserHTML()` on a `<span />`, which is required to
 // avoid the UI glitch so we configure our own sanitize function to make sure
 // it is safe to use `<span />`.
-const sanitizeNoticeHTML = (text: string): string => {
```

Comment:
Just wondering why you removed this?

Reply:
because the return type was wrong (it should be an object with a `__html` prop)